nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -349,6 +380,82 @@ in {
         Windows can unseal the encryption key.
       '';
     };
+
+    windows = mkOption {
+      default = { };
+      description = ''
+        Make Windows bootable from systemd-boot. This option is not necessary when Windows and
+        NixOS use the same EFI System Partition (ESP). In that case, Windows will automatically be
+        detected by systemd-boot.
+
+        However, if Windows is installed on a separate drive or ESP, you can use this option to add
+        a menu entry for each installation manually.
+
+        The attribute name is the title of the menu entry.
+      '';
+      example = literalExpression ''
+        {
+          "Windows 10".efiDeviceHandle = "HD0c3";
+          "Windows 11... Are you sure about this?" = {
```

Comment:
Maybe this should be more neutral, considering Windows 10 is going to go EOL next year (regular) / in two years (LTS)

Reply:
I wanted to have a little fun, but I see where you're coming from. I toned it down a little bit now, what do you think?

Reply:
I would prefer to make it completely neutral. We have all kinds of people using nixpkgs, and iirc windows support is actually being worked on. So I think that these things can come across as ostracising to windows users.

Reply:
Alright, I think both the example and test cases are neutral now :)